source/AsyncRgbLedAnalyzerSettings.cpp

```diff
@@ -67,7 +75,33 @@ const char* AsyncRgbLedAnalyzerSettings::SaveSettings()
 	SimpleArchive text_archive;
 
 	text_archive << mInputChannel;
-	text_archive << mBitRate;
+	text_archive << mLEDController;
 
 	return SetReturnString( text_archive.GetString() );
 }
+
+
+U8 AsyncRgbLedAnalyzerSettings::BitSize() const
+{
+	if (mLEDController == LED_LPD1886_12bit)
```

Comment:
the pattern here should probably match with the pattern in LEDChannelCount, either using a switch statement or an if statement.

Reply:
Okay, will make it a switch i think.